src/main/java/loanbook/commons/core/Messages.java

```diff
@@ -12,4 +12,16 @@
     public static final String MESSAGE_INVALID_PASSWORD = "The password provided is incorrect!";
     public static final String MESSAGE_INVALID_OLD_PASS = "The old password entered is incorrect!";
     public static final String MESSAGE_SAME_AS_CURRENT_PASSWORD = "The password provided is same as before!";
+    public static final String MESSAGE_AUTHEN_FAILURE = "Your password might be wrong "
+            + "or you did not enable \"less secure app\" in your google account setting before you use remind command."
+            + " Please refer to the User Guide!";
```

Comment:
Perhaps guide the user to the user guide? Please refer to the user guide by pressing the F1 key.

Reply:
Fixed.